README.md

```diff
@@ -27,7 +27,16 @@ Demonstrate calling the method, passing in "young prince" as the argument.
 
 Write your code here:
 ```ruby
-# code here
+def initialize (person)
+  @person = person
+end
+
+def offer_rose
+puts "would you take this rose, #{person}, in exchange for giving an old begger woman shelter from the bitter cold?"
+end
+
+Person.new('young prince')
+
```

Comment:
i think you're really close, you don't need any kind of initialize function, you just need an argument in your `offer_rose` method

```ruby
def offer_rose person
  puts "would you take this rose, #{person}, in exchange for giving an old begger woman shelter from the bitter cold?"
end
```